lib/shared/permissions.js

```diff
@@ -145,22 +146,13 @@ exports.elevateCommand = (command, options) => {
   const prefixedCommand = _.concat(exports.getEnvironmentCommandPrefix(options.environment), command);
 
   if (os.platform() === 'win32') {
-    const elevator = Bluebird.promisifyAll(require('elevator'));
-
-    return elevator.executeAsync(prefixedCommand, {
-      hidden: true,
-      terminating: true,
-      doNotPushdCurrentDirectory: true,
-      waitForTermination: true
-    }).then(() => {
+    const elevator = Bluebird.promisifyAll(bindings.load('elevator'));
```

Comment:
Would it be worth me adapting my CI bash script which checks all the `require` calls, to also check all the `bindings.load` calls?